sdk/src/Services/S3/Custom/_bcl/IAmazonS3.cs

```diff
@@ -0,0 +1,333 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using System.Threading;
+
+using Amazon.S3.Model;
+namespace Amazon.S3
+{
+#pragma warning disable CS1591
+    public partial interface IAmazonS3
+    {
+        #region GetACL
+        [Obsolete("GetACL combines both GetBucketAcl and GetObjectAcl and is deprecated. Please use the separated GetObjectAcl and GetBucketAcl operations.")]
+        Task<GetACLResponse> GetACLAsync(string bucketName, System.Threading.CancellationToken cancellationToken = default(CancellationToken));
```

Comment:
This method is missing documentation. Also, since this is the `bcl` folder, shouldn't you also have to include the non-async methods?

Reply:
good call, i did a manual test and the non-async methods were callable so I thought I was in the clear, but based on the pattern the other methods follow it looks like I missed adding it here.